Swift/QtUI/QtXMLSenderWidget.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2010-2016 Isode Limited.
+ * All rights reserved.
+ * See the COPYING file for more information.
+ */
+
+#include <Swift/QtUI/QtXMLSenderWidget.h>
+
+#include <string>
+#include <QString>
```

Comment:
We tend to group our includes, so ideally the QString should be grouped with other Qt includes. There is also a script doing this for you, see the python script https://github.com/swift/swift/blob/master/BuildTools/FixIncludes.py .